charts/docker-hub-rate-limit-exporter/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v2
 name: docker-hub-rate-limit-exporter
 description: Export your current docker hub rate limit status to prometheus.
+icon: https://github.com/shpwrck/docker-hub-rate-limit-exporter/raw/main/chart/icon.png
```

Comment:
very cool icon, thanks :-) mind changing the path so that the icon is loaded from this repository? That would be https://github.com/viadee/docker-hub-rate-limit-exporter/raw/main/chart/icon.png

Reply:
Can do!